app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupMakingSettingsViewHolder.java

```diff
@@ -70,4 +74,32 @@ private void applyNumberOfGroups() {
         }
         numGroups.clearFocus();
     }
+
+    @OnTextChanged(value = R.id.num_of_names_per_group, callback = OnTextChanged.Callback.AFTER_TEXT_CHANGED)
+    void afterNamesPerGroupChanged() {
+        String namesNumber = namesPerGroup.getText().toString().trim();
+        if (namesPerGroup.isFocused() && isValid(namesNumber)) {
+            numGroups.setText(getOffset(namesNumber));
+        }
+        dialog.getActionButton(DialogAction.POSITIVE).setEnabled(isValid(namesNumber));
+    }
+
+    @OnTextChanged(value = R.id.num_of_groups, callback = OnTextChanged.Callback.AFTER_TEXT_CHANGED)
+    void afterGroupNumChanged() {
+        String groupsNumber = numGroups.getText().toString().trim();
+        if (numGroups.isFocused() && isValid(groupsNumber)) {
+            namesPerGroup.setText(getOffset(groupsNumber));
+        }
+        dialog.getActionButton(DialogAction.POSITIVE).setEnabled(isValid(groupsNumber));
```

Comment:
Can we move this into the if statement?

Reply:
This will make it true and it will never get false.